docs/reference/commandline/dockerd.md

```diff
@@ -828,6 +829,24 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
+#### Air-gapped registries
```

Comment:
I think "registry on an air-gapped network" is better working. 71b47f59788ddbc05a97bde609054ce7070b2e3b switches to it.